.github/actions/spelling/allow.txt

```diff
@@ -1,3 +1,4 @@
+cidata
```

Comment:
I believe this should go into `expect.txt` instead of `allow.txt`?

Reply:
I guess, but doesn't that apply to (almost) everything else in `allow.txt` as well?

I can't really detect any pattern in what went into `allow.txt` except that most stuff went into `except.txt`, so maybe that should be the default location.

So I can move `cidata` to `except.txt`, but what about the rest of `allow.txt`?